### ISSUE TITLE
[BUG] - <title> tower does not work

### ISSUE THREAD

**Issue by AeroFoxxo** [Nexyl-Roblox/Issues#24]
### Description

tower do not work

### Reproduction steps

```bash
1. go into game
2. spawn riflemen (only one i have i cant try with others)
3. watch them both do nothing and stand there
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
https://imgur.com/a/sqkQMv6 staring with rifle in hip
```

### Logs

```bash
for future info how do i get logs
```

**Comment by AeroFoxxo** [comment]
is this just youguys turning off towers for now if not then i think its big severity

**Comment by gavinostler** [comment]
Towers were fixed as of #22
